packages/http/test/routes.test.ts

```diff
@@ -545,4 +551,51 @@ describe("http: routes", () => {
       { verb: "put", path: "/things/{thingId}", params: ["thingId"] },
     ]);
   });
+
+  describe("shared routes", () => {
```

Comment:
Feels like we should have a shared autoRoute unit test as well, in the rest library tests

Reply:
Yep, will add it.

Reply:
Done,